product_docs/docs/pem/9/certificates/index.mdx

```diff
@@ -0,0 +1,361 @@
+
+   ```shell
+   openssl x509 -req -days 365 -in agent.csr -CA ca_certificate.crt -CAkey ca_key.key -CAcreateserial -out agent.crt
+   ```
+
+1. Change the permission on the `agent.crt` and `agent.key` file:
+
+   ```shell
+   chmod 600 agent.crt agent.key
+   ```
+
+1. Use this agent certificate and key pair:
+
+   - for registering the new PEM agent from remote host to the PEM server.
+
+     a. Copy the agent certificate and key pair to the remote agent host and register the agent:
+
+        ```shell
+        export PEM_SERVER_PASSWORD=edb
+
+        /usr/edb/pem/agent/bin/pemworker --register-agent \
+        --pem-server 192.168.99.130 \
+        --pem-user enterprisedb \
+        --pem-port 5444 \
+        --pem-agent-user pem_agent_common_user \
+        -o agent_ssl_crt= agent.crt \
+        -o agent_ssl_key= agent.key
+        ```
+
+     b. Enable and start the pemagent services:
+
+        ```shell
+        systemctl enable pemagent
+        systemctl start pemagent
+        ```
+
+   - to replace the agent certificate and key pair with the registered agent.
+ 
+     a. Edit the `agent_user`, `agent_ssl_key`, and `agent_ssl_crt` parameters in `agent.cfg` file of the agent host
+
+        ```shell
+        vi /usr/edb/pem/agent/etc/agent.cfg
+        # Edit the agent username
+        agent_user=pem_agent_common_user
+        # Edit the ssl parameters with new certificate and key file location
+        agent_ssl_key=<new_location>/agent.key
+        agent_ssl_crt=<new_location>/agent.crt
+        ```
+      
+     b. Restart the pemagent service:
+
+        ```shell
+        systemctl restart pemagent
+        ```
+
+
+## Use certificates and keys signed by trusted CA
+
+Replace the PEM self-signed SSL certificates and keys with certificates and keys signed by trusted CA. 
+
+After obtaining the trusted CA certificates and keys replace the [server](/pem/latest/certificates/#replace-the-self-signed-server-ssl-certificates-with-the-certificates-signed-by-the-trusted-ca) and [agent](/pem/latest/certificates/#replace-the-self-signed-agent-ssl-certificates-with-the-certificates-signed-by-the-trusted-ca) certificates and keys.
+
+### Replace the self-signed server SSL certificates with the certificates signed by the trusted CA
+
+1. Take the backup of old server certificate and key files:
+
+   ```shell
+   # Running as root
+   mkdir /var/lib/edb/as<x>/data/certs
+   cd /var/lib/edb/as<x>/data/
+   mv server.* root.* ca_* /var/lib/edb/as<x>/data/certs
+   ```
+
+1. Generate a private key for the server:
+
+   ```shell
+   openssl genrsa -out server.key 4096 
+   ```
+
+1. Generate a Certificate Signing Request (CSR) for the server:
+
+   ```shell
+   openssl req -new -key server.key -out server.csr -subj '/C=IN/ST=MH/L=Pune/O=EDB/CN=PEM'
+   ```
+   
+   Where `-subj` is provided as per your requirements. We recommend using the `Hostname/Domain Qualified Full Name` of the PEM server host for `CN`.
+   
```

Comment:
Not sure why Hostname/Domain Qualified Full Name is in in caps and tagged as code. I think you just want to say:
hostname or domain-qualified full name
with no special tagging